nncf/common/graph/transformations/commands.py

```diff
@@ -63,6 +63,8 @@ class TargetType(IntEnum):
                           specific module attributes - N/A in TF
     `OPERATOR_POST_HOOK` - a location after a function call in PT without access to
                            specific module attributes - N/A in TF
+    `POST_BRANCH_WITH_PARTIAL_MERGE` - a location after the associated model layer,
```

Comment:
Did you implement support for POST_BRANCH_WITH_PARTIAL_MERGE in the ONNX and Torch?

Reply:
This is not applicable to the Torch as I understand.
For the ONNX it's implemented as well.

Reply:
Please, open a ticket for Torch implementation.